mne/io/base.py

```diff
@@ -1361,12 +1363,17 @@ def filter(self, l_freq, h_freq, picks=None, filter_length='auto',
         # Deal with annotations
         onsets, ends = _annotations_starts_stops(
             self, skip_by_annotation, 'skip_by_annotation', invert=True)
-        for start, stop in zip(onsets, ends):
+        logger.info('Filtering raw data in %d contiguous segment%s'
+                    % (len(onsets), _pl(onsets)))
+        for si, (start, stop) in enumerate(zip(onsets, ends)):
+            if si > 0:
+                verbose = 'warning' if verbose != 'error' else verbose
```

Comment:
I am not sure I understand. You have an info all the time and then a verbose you pass to filter. Why?